src/plugins/recordTypes/uoc/forms/default.jsx

```diff
@@ -0,0 +1,74 @@
+import { defineMessages } from 'react-intl';
+
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    // Col,
+    // Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="useOfCollections" collapsible>
+
```

Comment:
The tab order of fields doesn't look like it follows the style guide: https://github.com/collectionspace/cspace-ui.js/tree/master/docs/style#fields

Reply:
Interesting.  I'll need to check with Megan.  She may want to change the layout if we can't to follow that style guide